module_9/task1.php

```diff
@@ -0,0 +1,17 @@
+<pre>
+<?php
+
+$line = 'Student, hello!';
+$control = [];
+$lenghtOfLine = strlen($line);
+
+for ($i = 1; $i <= $lenghtOfLine; $i++) {
+    $tmp = substr($line, strlen($line) - 1);
+    $line = substr($line, 0, -1);
+    $control[$tmp]++;
```

Comment:
```
PHP Notice:  Undefined index: ! in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: o in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: l in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: e in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: h in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index:   in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: , in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: t in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: n in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: d in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: u in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
PHP Notice:  Undefined index: S in C:\Users\user\Desktop\PHP\homework\module_9\task1.php on line 10
```
I've got these errors during running your code

Reply:
@Lehersun I dont understand the reason of these mistakes. Can i use function error_reporting() in same situations?

Reply:
It's because of  incrementing not set value in array $control[$tmp]++;
You have to make it another way